operator/internal/controller/paladin_controller.go

```diff
@@ -437,7 +531,7 @@ func (r *PaladinReconciler) createPostgresPVC(ctx context.Context, node *corev1a
 }
 
 func (r *PaladinReconciler) addKeystoreSecretMounts(ss *appsv1.StatefulSet, signers []corev1alpha1.SecretBackedSigner) {
-	paladinContainer := &ss.Spec.Template.Spec.Containers[0]
+	paladinContainer := &ss.Spec.Template.Spec.Containers[1] // TODO this changes depending on PG mode
```

Comment:
TODO: need to make this index conditional / contexted so that we can determine if its `1` or `0` depending of PG sidecare mode is enabled.

Reply:
Discussed with @dwertent - this code should just "find" the container rather than guess - in case we add any future sidecars (hopefully spinning the UI out separately).